start.sh

```
@@ -1,2 +1,3 @@
-npm run build
-npm run start
+pnpm install
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)